CODE_OF_CONDUCT.md

```diff
@@ -26,47 +26,54 @@ ambiente positivo per la nostra comunità:
+
+Attualmente gli Ambassador appartenenti al gruppo Discussion che hanno la responsabilità in prima persona di questo regolamento e della gestione dei casi sono:
+Alessandro Cappellozza
+Simone Gizzi
+Andrea Guzzo
+
+## Soggetti
+
+Questo Codice di Comportamento si applica a chiunque faccia parte della community sia che la rappresenti o che partecipi a qualsiasi titolo nei contenuti, nei commenti, nelle discussioni, e in generale in qualsiasi manifestazione correlata come talk o meetup.
 
-Questo Codice di Comportamento si applica sia all'interno degli spazi della
-comunità che negli spazi pubblici quando un individuo rappresenta la sua
-comunità. Esempi di rappresentanza della nostra comunità includono l'uso di un
-indirizzo e-mail ufficiale del progetto, la pubblicazione tramite un account
-ufficiale attraverso social media o la funzione di rappresentante designato ad
-un evento online o offline.
+Esempi di rappresentanza della nostra comunità includono la comunicazione online su Telegram e su GitHub, la pubblicazione tramite un account ufficiale attraverso social media o la funzione di rappresentante designato ad un evento online o offline.
 
 ## Applicazione
 
-I casi di comportamento abusivo, molesto o altrimenti inaccettabile
-possono essere presentati contattando i responsabili dell'applicazione di questo
-Codice di Condotta della comunità commentanto la [discussione apposita](https://github.com/Il-Libro-Open-Source/book/discussions/30).
-.Tutti i reclami saranno prontamente esaminati ed indagati.
+I casi di comportamento abusivo, molesto o altrimenti inaccettabile che non rispettano le linee guida possono essere presentati con due modalità:
+Modalità pubblica: attraverso il commento dell’apposita discussione su [Github](https://github.com/Il-Libro-Open-Source/book/discussions/30). Per facilitare l’intervento e la supervisione è consigliato il tag di un responsabile della comunità (Ambassador).
+Modalità privata: è possibile compilare il [seguente google form privato](https://forms.gle/xpr46LzcJ6wzPtEQ8) che verrà visionato solamente dal gruppo di Ambassadors responsabile del gruppo Discussion.
 
-Tutte le guide della comunità sono obbligate a mantenere la riservatezza della
-persona che ha riportato il caso.
+Tutti i reclami saranno prontamente esaminati, indagati e presi in carico dai responsabili della comunità.
+Tutte le persone responsabili della comunità e che prenderanno in carico la supervisione di uno specifico caso sono obbligate a mantenere la riservatezza della persona che ha riportato il caso.
 
 ## Linee guida all'applicazione
 
 Le guide della comunità seguiranno queste linee guida per determinare le
-azioni correttive contro azioni che ritengono violare questo Codice di Condotta:
+azioni correttive contro azioni che ritengono violare questo Codice di Condotta.
```

Comment:
Hai usato codice di comportamento in tutto il documento, manteniamo questo termine

```suggestion
azioni correttive contro azioni che ritengono violare questo Codice di Comportamento.
```